storm-core/src/jvm/org/apache/storm/executor/Executor.java

```diff
@@ -204,8 +204,8 @@ public static Executor mkExecutor(Map workerData, List<Long> executorId, Map<Str
                 throw Utils.wrapInRuntime(ex);
             }
         }
-        executor.init(idToTask);
 
+        executor.idToTask = idToTask;
```

Comment:
Yes, actually the code style is more likely to clojure itself indeed. It's needed for ease of port work, and we can address later.